release_docs/RELEASE.txt

```diff
@@ -47,12 +47,6 @@ New Features
 
     Configuration:
     -------------
-    - Added signed Windows msi binary and signed Apple dmg binary files.
-
-      The release process now provides signed Windows and Apple installation
-      binaries in addition to the debian and rpm installation binaries. Also
-      these installer files are no longer compressed into packaged archives.
-
     - Added configuration option for internal threading/concurrency support:
```

Comment:
Should this be removed too?

Reply:
develop and 1.16 are initially the same thing so the Release.txt file should be the same

Reply:
Therefore anything that did not go into 1.14 releases need to be in both 1.16 and develop.